matlabCode/createBIDS_anat_Full_T1w_json.m

```diff
@@ -1,6 +1,8 @@
 %% Template Matlab script to create an BIDS compatible sub-01_ses-01_acq-FullExample_run-01_T1w.json file
 % This example lists all required  RECOMMENDED and optional fields.
 % When adding additional metadata please use CamelCase 
+%
+% Writing json files relies 
```

Comment:
Is this sentence incomplete?

Reply:
Yup... My train of thought derailed. Was about to start putting some comments on which library need to be used for the script to work but I figured that we should first agree on that before putting the comments.